scene/gui/graph_edit.cpp

```diff
@@ -984,12 +979,12 @@ void GraphEdit::_gui_input(const Ref<InputEvent> &p_ev) {
 
 		if (b->get_button_index() == BUTTON_WHEEL_UP && b->is_pressed()) {
 			//too difficult to get right
-			//set_zoom(zoom*ZOOM_SCALE);
+			//set_zoom(zoom*zoom_step);
```

Comment:
It would be better to remove this as it's dead code.

Reply:
Obsolete, as this code is no longer dead in master.